src/DynamoCoreWpf/Views/Core/AnnotationView.xaml.cs

```diff
@@ -33,8 +33,6 @@ public AnnotationView()
             Resources.MergedDictionaries.Add(SharedDictionaryManager.DynamoColorsAndBrushesDictionary);
             Resources.MergedDictionaries.Add(SharedDictionaryManager.DataTemplatesDictionary);
             Resources.MergedDictionaries.Add(SharedDictionaryManager.DynamoConvertersDictionary);
-            Resources.MergedDictionaries.Add(SharedDictionaryManager.InPortsDictionary);
-            Resources.MergedDictionaries.Add(SharedDictionaryManager.OutPortsDictionary);
```

Comment:
Have these dictionaries been removed or has the code in the dictionaries been moved to the code behind code in C#?